draft-ietf-tls-tls13.md

```diff
@@ -3131,7 +3185,8 @@ aspect of the certificate chain was unacceptable (e.g., it was not signed by a
 known, trusted CA), the server MAY at its discretion either continue the
 handshake (considering the client unauthenticated) or abort the handshake.
 
-Any endpoint receiving any certificate signed using any signature algorithm
+Any endpoint receiving any certificate that must be validated and is
+signed using any signature algorithm
```

Comment:
This section here is on how to handle the receipt of a certificate message and its contents, not how to handle the entirety of a certificate chain, including root store. A self-signed root using MD5 would generally not need to be included in the certificate chain sent to the client, and thus would not actually hit the existing text.
https://tools.ietf.org/html/draft-ietf-tls-tls13-19#page-59

I don't really want clients accepting use of MD5 in any scenario, but adding a note restating that these checks do not apply to a trust anchor not sent in this message would be fine. This would not change the existing requirements; just clarify them. Yeah, there's a valid argument to be made that the same checks should apply to all certs regardless of where the client gets them, but the difference here is that this is something that goes through the TLS protocol, specifically (and I don't really want to reopen the discussion at all, though, please do so on list if you disagree).

(Just because it's not clear looking at GitHub here, this PR was merged without this specific changeset.)